### PR TITLE
Add LaserScan visualization plugin

### PR DESCRIPTION
**Pull request by Sarath18**
* Create laser scan visualization plugin for ign-rviz

* Add laser scan display option in custom drawer menu in the main application

Signed-off-by: Sarathkrishnan Ramesh <sarathkrishnan99@gmail.com>

Reference to issue #17

### CODE REVIEW

**Comment by Sarath18** [comment]
LaserScan visualization plugin preview

laser_scan

File: ign_rviz/include/ignition/rviz/rviz.hpp [hunk 1]
```diff
@@ -95,6 +116,7 @@ class RViz : public QObject
 
   // Plugins
   std::shared_ptr<plugins::MessageDisplay<tf2_msgs::msg::TFMessage>> tf_plugin;
+  std::shared_ptr<plugins::MessageDisplay<sensor_msgs::msg::LaserScan>> laser_scan_plugin;
```
Comment (ahcorde) [inline ign_rviz/include/ignition/rviz/rviz.hpp:119]: You are defining here just one laser scan plugin, but we should be able to create more than one, as well as be able to remove it if we don't want to use anymore. Can you think about a data structure to keep this data?

File: ign_rviz_plugins/src/rviz/plugins/laser_scan_display.cpp [hunk 2]
```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2020 Open Source Robotics Foundation, Inc.
```
Comment (ahcorde) [inline ign_rviz_plugins/src/rviz/plugins/laser_scan_display.cpp:1]: This class should be replaced with the LidarVisual when it's ready. Can you create a ticket for this?

File: ign_rviz/include/ignition/rviz/rviz.hpp (hunk 1 above)
Comment (Sarath18) [reply ign_rviz/include/ignition/rviz/rviz.hpp:119]: We can use `std::vector` of plugin shared pointers?

File: ign_rviz_plugins/src/rviz/plugins/laser_scan_display.cpp (hunk 2 above)
Comment (Sarath18) [reply ign_rviz_plugins/src/rviz/plugins/laser_scan_display.cpp:1]: Created a new ticket #19

**Pull request merged**